buildenvs/github-action.Dockerfile

```diff
@@ -95,7 +98,14 @@ RUN ln -s /usr/bin/cpp-12                                   /usr/bin/cc; \
 
 WORKDIR /github/workspace
 
-RUN useradd -rm -d /home/runner -s /bin/bash -g root -G sudo -u 1001 runner
+RUN sed -i "s/999/1001/g" /etc/login.defs && \
+    groupadd -g 127 runner && \
+    useradd -rm -d /home/runner -s /bin/bash -g runner -G sudo -u 1001 runner && \
+    sed -i /etc/sudoers -re 's/^%sudo.*/%sudo ALL=(ALL:ALL) NOPASSWD: ALL/g' && \
+    sed -i /etc/sudoers -re 's/^root.*/root ALL=(ALL:ALL) NOPASSWD: ALL/g' && \
+    sed -i /etc/sudoers -re 's/^#includedir.*/## **Removed the include directive** ##"/g' && \
+    echo "runner ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+    chown -R runner:runner /github/workspace;
```

Comment:
Can you switch this to use the `; \` syntax vs. the `&& \` syntax by prepending `set -xe; \` to the top of this sequence.